docs/install/install_options/install_options.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Options
+---
+
+This page focuses on the options that can be used when you set up RKE2 for the first time:
+
+- [Options for installation with script](#options-for-installation-with-script)
+- [Options for running from binary](#running-rke2-from-the-binary)
+- [Registration options for the RKE2 server](#registration-options-for-the-rke2-server)
+- [Registration options for the RKE2 agent](#registration-options-for-the-rke2-agent)
+- [Configuration File](#configuration-file)
+
+In addition to configuring RKE2 with environment variables and CLI arguments, RKE2 can also use a [config file.](#configuration-file)
+
+> Throughout the RKE2 documentation, you will see some options that can be passed in as both command flags and environment variables. For help with passing in options, refer to [How to Use Flags and Environment Variables.](./how_to_flags/how_to_flags.md)
+
+### Options for Installation with Script
+
+As mentioned in the [Quick-Start Guide](../../install/quickstart.md), you can use the installation script available at https://get.rke2.io to install RKE2 as a service.
+
+The simplest form of this command is as follows:
+```sh
+curl -sfL https://get.rke2.io | sh -
+```
+
+When using this method to install RKE2, the following environment variables can be used to configure the installation:
+
+| Environment Variable | Description |
+|-----------------------------|---------------------------------------------|
+| `INSTALL_RKE2_VERSION` | Version of RKE2 to download from GitHub. Will attempt to download from the latest channel if not specified. |
+| `INSTALL_RKE2_TYPE` | Type of systemd service to create, can be either "server" or "agent" Default is "server". |
+| `INSTALL_RKE2_CHANNEL_URL` | Channel URL for fetching RKE2 download URL. Defaults to https://update.rke2.io/v1-release/channels. |
+| `INSTALL_RKE2_CHANNEL` | Channel to use for fetching RKE2 download URL. Defaults to "latest". Options include: `stable`, `latest`, `testing`. |
+| `INSTALL_RKE2_METHOD` | Method of installation to use. Default is on RPM-based systems "rpm", all else "tar". |
+
+
+Environment variables which begin with `RKE2_` will be preserved for the services to use.
```

Comment:
no they wont

Reply:
thanks. I will remove that line.

Reply:
Or you meant I should warn that they will not be preserved?